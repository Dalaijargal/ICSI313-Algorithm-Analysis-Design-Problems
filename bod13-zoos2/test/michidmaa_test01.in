1000000 6
11 25 36 61 97 158