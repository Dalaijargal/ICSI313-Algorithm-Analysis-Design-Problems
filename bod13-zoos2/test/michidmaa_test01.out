901251734179353791